client/src/pages/HomePage/HomePage.js

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
+import CardContainer from "../../components/CardContainer/CardContainer";
```

Comment:
Suggest adding CardContainer component to `index.js` in `./components` and import this component from there